src/components/layout/Undernames/Undernames.tsx

```diff
@@ -0,0 +1,188 @@
+import { Pagination, Tooltip } from 'antd';
+import Table from 'rc-table';
+import { ColumnType } from 'rc-table/lib/interface';
+import { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { useArweaveCompositeProvider, useIsMobile } from '../../../hooks';
+import { useUndernames } from '../../../hooks/useUndernames/useUndernames';
+import {
+  ArweaveTransactionID,
+  PDNTContractJSON,
+  UndernameMetadata,
+} from '../../../types';
+import { isArweaveTransactionID } from '../../../utils';
+import { ArrowLeft } from '../../icons';
+import Loader from '../Loader/Loader';
+
+function Undernames() {
+  const arweaveDataProvider = useArweaveCompositeProvider();
+  const isMobile = useIsMobile();
+  const navigate = useNavigate();
+  const { id } = useParams();
+  if (!id) {
+    navigate(-1);
```

Comment:
going back -1 could return a user to an external page. instead have this route to `'manage/pdnts'`